### ISSUE TITLE
"BETWEEN ? AND ?" not working in SQLite

### ISSUE THREAD

**Issue by TeddyDD** [kyleconroy/sqlc#1958]
### What do you want to change?

Same request as #1077, but for SQLite 

Dumb example:
```
-- name: ListAuthorBet :many
SELECT * FROM authors
WHERE id BETWEEN ? AND ?;
```
Results in 

```go
func (q *Queries) ListAuthorBet(ctx context.Context) ([]Author, error) {
...
```

Tested on current main (3623ec87fd3015f00cdc9e2c29c45d5557a12473)

**Comment by mvdan** [comment]
Is this fixed by #1967?

**Comment by TeddyDD** [comment]
Yes
